Support/lib/rspec/mate/switch_command.rb

```diff
@@ -140,8 +140,9 @@ def klass(relative_path, content=nil)
       def write_and_open(path)
         FileUtils.mkdir_p(File.dirname(path))
         described = described_class_for(path, base_dir)
+        is_rails = File.directory? path.sub(%r{/spec/.*$}, '/app')
```

Comment:
👍🏻 Fine by me. Ideally I think there should be a single (reliable?) method dedicated to rails detection. 

(I'll push the change later)